multiversx_sdk_cli/cli_deps.py

```diff
@@ -34,16 +36,63 @@ def install(args: Any):
 
 def check(args: Any):
     name: str = args.name
-    module = dependencies.get_module_by_key(name)
-    tag_to_check: str = config.get_dependency_tag(module.key)
+
+    if name == "all":
```

Comment:
After re-thinking about this, maybe we can drop `check all` and `install all`? On the long run, maybe it's better (less issues, less corner cases) if we do.

Let's double check with @ovidiuolteanu and @ccorcoveanu. I feel `deps install all` is quite exotic.